libs/ptg-ui-web-components/src/components/ptg-tooltip/ptg-tooltip.scss

```diff
@@ -0,0 +1,222 @@
+}
+.tooltip .top {
+  left: 40%;
+  top: 10px;
+  display: none;
+  transform: translate(0, -50%);
+  transform: translate(-30%, -100%);
+  -webkit-transform: translate(-30%, -100%);
+}
+.tooltip .top i {
+  top: 100%;
+  width: 30px;
+  height: 15px;
+  margin-left: -15px;
+  left: 46%;
+  // margin-left: -60px;
+}
+.tooltip .top i::after {
+  left: 50%;
+  transform: translate(-50%, -50%) rotate(45deg);
+}
+.tooltip .right {
+  top: 50%;
+  left: 100%;
+  opacity: 0;
+  margin-left: 20px;
+  visibility: hidden;
+  transition: opacity 0.8s;
+  transform: translate(0, -50%);
+}
+.tooltip:hover .right {
+  opacity: 1;
+  visibility: visible;
+}
+.tooltip .right i {
+  top: 50%;
+  right: 100%;
+  width: 12px;
+  height: 24px;
+  // overflow: hidden;
+  margin-top: -12px;
+  // position: absolute;
+}
+.tooltip .right i::after {
+  left: 0;
+  top: 50%;
+  // content: '';
+  // width: 12px;
+  // height: 12px;
+  // position: absolute;
+  // background-color: black;
+  transform: translate(50%, -50%) rotate(-45deg);
+}
+.tooltip .bottom {
+  left: 50%;
+  opacity: 0;
+  padding: 3px;
+  visibility: hidden;
+  transition: opacity 0.8s;
+  transform: translate(-50%, 0);
+}
+.tooltip .shadow {
+  box-shadow: 0 1px 8px rgba(0, 0, 0, 0.5);
+}
+.tooltip:hover .bottom {
+  opacity: 1;
+  display: block;
+  visibility: visible;
+}
+.tooltip .bottom i {
+  left: 40%;
+  width: 24px;
+  height: 12px;
+  bottom: 100%;
+  margin-top: -15px;
+}
+.tooltip .bottom i::after {
+  left: 50%;
+   transform: translate(-50%, 50%) rotate(45deg);
+}
+p{
+  cursor: pointer;
+}
+
+@media only screen and (max-width: 768px) {
+  /* For mobile phones: */
+  .tooltip .right {
+    top:100%;
+    left: 40%;
+    opacity: 0;
+    padding: 3px;
+    visibility: hidden;
+    transition: opacity 0.8s;
+    transform: translate(-50%, -12px);
+  }
+ 
+  .tooltip:hover .right {
+    opacity: 1;
+    display: block;
+    visibility: visible;
+  }
+
+  .tooltip .right i {
+    left: 40%;
+    width: 24px;
+    height: 22px;
+    bottom: 100%;
+    margin-top: -48px;
+    overflow: hidden;
+    position: absolute;
+    
+  }
+  .tooltip .right i::after {
+    content: '';
+    width: 12px;
+    height: 12px;
+    position: absolute;
+    background-color: #000;
+    left: 50%;
+     transform: translate(-50%, 50%) rotate(45deg);
+  }
+  .tooltip .left {
+    left: 40%;
+    opacity: 0;
+    padding: 3px;
+    visibility: hidden;
+    transition: opacity 0.8s;
+    transform: translate(-50%, 17px);
+  }
+ 
+  .tooltip:hover .left {
+    opacity: 1;
+    display: block;
+    visibility: visible;
+  }
+
+  .tooltip .left i {
+    left: 40%;
+    width: 24px;
+    height: 21px;
+    bottom: 100%;
+    margin-top: -46px;
+    overflow: hidden;
+    position: absolute;
+    
+  }
+  .tooltip .left i::after {
+    content: '';
+    width: 12px;
+    height: 12px;
+    position: absolute;
+    background-color: #000;
+    left: 50%;
+     transform: translate(-50%, 50%) rotate(45deg);
+  }
+}
```

Comment:
@poojacsawai Update the whole file to have styles in the hierarchy. This means keeping the CSS in a nested way, taking the benefits of scss instead of writing CSS only.